src/Focusable.ts

```diff
@@ -203,6 +207,10 @@ export class FocusableAPI implements Types.FocusableAPI {
             onUncontrolled,
         } = options;
 
+        const onElement = (
+            options as Types.FindSomeProps | { onElement: undefined }
```

Comment:
why can't this option be supported by all `find` APIs ?

Reply:
It makes no sense for methods that find single element (i.e. for everything but `findAll`). I can add it to `findAll`. The new method supports finding elements from a certain element forward or back. `findAll` name doesn't feel exactly correct semantically because of «all». We can remove both `findSome` and `findAll` and have just one `findElements` method (or just `find`). But that would require major version increment. Are we currently using `findAll` in FUI?

Reply:
I will move `onElement` to `FindFocusableProps`. `FindFirstProps`, `FindNextProps` and others use pick to get props from there anyway. But it still makes no sense to pick it for a single element search methods.

Reply:
Ok, I'll go back to what I was doing initially — add it all to `findAll`.

Reply:
So, it all is a part of `findAll` now.